project_multifamily_beta/housing_characteristics/Dishwasher.tsv

```diff
@@ -1,5 +1,202 @@
-Dependency=Usage Level	Option=None	Option=318 Rated kWh, 80% Usage	Option=318 Rated kWh, 100% Usage	Option=318 Rated kWh, 120% Usage	Option=290 Rated kWh, 80% Usage	Option=290 Rated kWh, 100% Usage	Option=290 Rated kWh, 120% Usage
-Low	0.000000	0.360000	0.000000	0.000000	0.640000	0.000000	0.000000
-Medium	0.000000	0.000000	0.360000	0.000000	0.000000	0.640000	0.000000
-High	0.000000	0.000000	0.000000	0.360000	0.000000	0.000000	0.640000
-Average	0.000000	0.000000	0.360000	0.000000	0.000000	0.640000	0.000000
+Dependency=Geometry Building Type RECS	Dependency=Location Region	Dependency=Usage Level	Option=290 Rated kWh, 100% Usage	Option=290 Rated kWh, 120% Usage	Option=290 Rated kWh, 80% Usage	Option=318 Rated kWh, 100% Usage	Option=318 Rated kWh, 120% Usage	Option=318 Rated kWh, 80% Usage	Option=None	sample_count	sample_weight
+Mobile Home	CR02	Average	0.060507	0.000000	0.000000	0.213799	0.000000	0.000000	0.725693	7.250000	63468.816406
```

Comment:
Side note: Not necessary for this PR, but we can remove `Average`. It hasn't been used in a long time and is functionally the same as `Medium`.

Reply:
Thank you, I always wanted to do this.